src/frameworks/drupal7/solr/solr.md

```diff
@@ -0,0 +1,8 @@
+# Using Solr with Drupal 7.x
```

Comment:
This file doesn't really add much.  Let's just drop it and make the other two files top-level in this section.  Fewer pages for users to click through.